### ISSUE TITLE
[BUG: External]: Blank lines cause huge gap when Obsidian Columns is used inside Admonition codeblock

### ISSUE THREAD

**Issue by nousernameavailableanymore** [tnichols217/obsidian-columns#18]
# Describe the bug
Blank lines cause huge gap when Obsidian Columns is used inside Admonition codeblock

## Steps To Reproduce
1. Have Admonitions plugin installed
2. Create an admonition in a note
3. Put a col/col-md block inside the admonition codeblock
4. Type some text
5. Hit "Enter" twice to create a new line, then a blank line.
6. Type some more text (there should now be a blank line between the first and second text)

## Expected behavior
The two lines of text should have the same gap / distance as when using Obsidian Columns outside of Admonition codeblock

## Actual behavior
The two lines of text are considerably more apart.

## Additional info
Issue might be related to fix in https://github.com/tnichols217/obsidian-columns/issues/17

Testcode:
``````
`````ad-note
````col
```col-md
line1

line2
```
```col-md
line1

line2
```
````
`````
``````

## Screenshots
![image](https://user-images.githubusercontent.com/45376324/172033507-172267ed-45f1-44ae-84f9-676c9094679b.png)
![image](https://user-images.githubusercontent.com/45376324/172033557-103dd7bf-403c-4080-b7e9-b252cc017977.png)

## System Info
Obsidian Version: 0.14.6
Plugin Version:
- Admonition: 9.0.3 (latest)
- Obsidian Columns 1.1.5 (latest)

Btw:
All functionality based on block quotes (like the new callouts feature) is extremely laborious to use.
Its too complicated to paste text into it (as this does not retain / create the block quote characters), creating blank lines in the code is annoying and putting two SEPARATE Callouts right beneath each other (without a blank line in between) seems to be plain impossible.
Imho code block based functionality (like Obisdian Columns) is far superior to block quote based functionality.

**Comment by tnichols217** [comment]
Sorry this must've been caused by #17's fix, my bad I'll fix it in 3 hours.

**Comment by tnichols217** [comment]
Fixed in commit [7e48560](https://github.com/tnichols217/obsidian-columns/commit/7e485601dd021c8e1bb53f277062fd11f0f790d9), version [1.1.6](https://github.com/tnichols217/obsidian-columns/releases/tag/1.1.6)

**Comment by nousernameavailableanymore** [comment]
I updated to 1.1.6 but still same behavior.
I also tested in Default theme, same issue.

![image](https://user-images.githubusercontent.com/45376324/172265522-d1f191c5-a115-4830-8f8d-bb85e06d852f.png)
